File: Step4-Python/1110.py
num=int(input()) #초기값
cmp_num=num #비교값
count=0

while True:
    ten=cmp_num//10 #몫=십의자리  
    one=cmp_num%10 #나머지=일의자리  

    #두 수를 더해 result값 만들기
    result=ten+one
    
    #새로운 비교값만들기
    cmp_num=one*10+result%10

    count+=1 #사이클 계산
    #원래 값과 새로운값 비교
    if num==cmp_num:
        break

print(count)

File: Step4-C/1110.c
#define _CRT_SECURE_NO_WARNINGS
#include <stdio.h>
#include <stdbool.h>

int main()
{
	int num;
	scanf("%d", &num);

	int cmp_num = num;
	int count = 0;

	while (true)
	{
		int ten = cmp_num / 10;
		int one = cmp_num % 10;
		int result = ten + one;

		cmp_num = one * 10 + result % 10;

		count += 1;
		if (num == cmp_num)
			break;
	}

	printf("%d", count);

	return 0;
}